src/Apps/CollectorProfile/Routes/SavesAndFollows/CollectorProfileSavesAndFollowsRoute.tsx

```diff
@@ -11,15 +13,25 @@ interface CollectorProfileSavesAndFollowsRouteProps {
 const CollectorProfileSavesAndFollowsRoute: React.FC<CollectorProfileSavesAndFollowsRouteProps> = ({
   me,
 }) => {
+  const isArtworkListsEnabled = useFeatureFlag("force-enable-artworks-list")
+
+  if (isArtworkListsEnabled) {
+    return <CollectorProfileSaves2RouteFragmentContainer me={me} />
+  }
+
   return <CollectorProfileSavesRouteQueryRenderer />
 }
 
 export const CollectorProfileSavesAndFollowsRouteFragmentContainer = createFragmentContainer(
   CollectorProfileSavesAndFollowsRoute,
   {
     me: graphql`
-      fragment CollectorProfileSavesAndFollowsRoute_me on Me {
-        name
+      fragment CollectorProfileSavesAndFollowsRoute_me on Me
+        @argumentDefinitions(
+          shouldFetchArtworkListsData: { type: "Boolean!" }
+        ) {
+        ...CollectorProfileSaves2Route_me
+          @include(if: $shouldFetchArtworkListsData)
```

Comment:
we will fetch this data only if feature flag is enabled

Reply:
Cool, didn't know about this directive together with if!